docs/getting-started/with-clickhouse.mdx

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_position: 5
+sidebar_label: With ClickHouse
+description: "Learn the basics of Hasura DDN and how to get started with a ClickHouse database."
+keywords:
+  - hasura ddn
+  - graphql api
+  - getting started
+  - guide
+  - clickhouse
+---
+
+import Prereqs from "@site/docs/_prereqs.mdx";
+
+# Get Started with Hasura DDN and ClickHouse
+
+<Prereqs />
+
+## Guide
+
+In this guide, you'll learn how to get started with Hasura DDN and a ClickHouse database. You'll set up a new Hasura DDN
+project, connect it to a new ClickHouse database, and run your first query.
+
+### Step 1. Create a table in your ClickHouse database
+
+```sql title="First, you'll create a new table in your ClickHouse database:"
+--- Create the table
+CREATE TABLE users (
+  id UInt32,
+  name String,
+  age UInt8
+) ENGINE = MergeTree()
+ORDER BY id;
+
+--- Insert some data
+INSERT INTO users (id, name, age) VALUES (1, 'Alice', 25);
+INSERT INTO users (id, name, age) VALUES (2, 'Bob', 30);
+INSERT INTO users (id, name, age) VALUES (3, 'Charlie', 35);
+```
+
+:::tip Where's my database?
+
+Hasura DDN can connect to hosted or local data sources. If you have a cloud-hosted ClickHouse provider, you can connect
+to your database and seed the table with the above data. Alternatively, you can
+[run ClickHouse locally using Docker](https://hub.docker.com/r/clickhouse/clickhouse-server/).
+
+:::
+
+### Step 2. Authenticate your CLI
+
+```sh title="Before you can create a new Hasura DDN project, you need to authenticate your CLI:"
+ddn auth login
+```
+
+### Step 3. Scaffold out a new local project
+
+```sh title="Next, you'll create a new Hasura DDN project. Run the following command:"
+ddn supergraph init my-project && cd my-project
+```
+
+### Step 4. Initialize your ClickHouse connector
+
+```sh title="Initialize a new ClickHouse connector. Run the following command:"
+ddn connector init my_clickhouse -i
+```
+
+From the dropdown, start typing `ClickHouse` and hit enter.
+
+Set your environment variables:
+
+| Variable              | Value                                                 | Required |
+| --------------------- | ----------------------------------------------------- | -------- |
+| `CLICKHOUSE_URL`      | `clickhouse://username:password@hostname:port/dbname` | Yes      |
```

Comment:
This is incorrect. We use the http interface for clickhouse, so the url looks like:
`http(s)://hostname:port`, where port `8123` is the default when running locally. I think on cloud you get a random port assigned to you.